### PR TITLE
Oracle NetSuite Entities & Activities Sources documentation initial commit

### PR DESCRIPTION
**Pull request by mohan-h**

### CODE REVIEW

File: help/sources/connectors/marketing-automation/oracle-netsuite.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+## Prerequisites {#prerequisites}
+
+Before you can bring your [!DNL Oracle NetSuite] data to Experience Platform, you must first ensure that you have the following:
+
+* A [!DNL Oracle NetSuite] account. If you do not already have an account, contact [!DNL Oracle NetSuite](https://www.NetSuite.com/portal/company/contactus.shtml).
+* Have an active subscription to any product.
+* [!DNL Oracle NetSuite] uses OAuth 2.0 to communicate with [!DNL Oracle NetSuite] API's. 
+    * You will need your **Account ID**. Refer to the [Finding your NetSuite Account ID](https://docs.oracle.com/en/cloud/saas/netsuite/ns-online-help/section_1498754928.html#Finding-Your-NetSuite-Account-ID)] page if you do not know your NetSuite Account ID.
+    * You will need your **[!DNL NetSuite] Account ID**. Refer to the [Finding your NetSuite Account ID](https://docs.oracle.com/en/cloud/saas/netsuite/ns-online-help/section_1498754928.html#Finding-Your-NetSuite-Account-ID)] page if you do not know your NetSuite Account ID.
```
Comment (carlojavier) [inline help/sources/connectors/marketing-automation/oracle-netsuite.md]: I think we just need one of these bullet points. We can just keep the first one.

File: help/sources/connectors/marketing-automation/oracle-netsuite.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+## IP address allow list {#ip-allow-list}
+
+A list of IP addresses may require to be added to an allow list prior to working with source connectors. Failing to add your region-specific IP addresses to your allow list may lead to errors or non-performance when using sources. See the [IP address allow list](../../ip-address-allow-list.md) page for more information.
+
+## Prerequisites {#prerequisites}
+
+Before you can bring your [!DNL Oracle NetSuite] data to Experience Platform, you must first ensure that you have the following:
+
+* A [!DNL Oracle NetSuite] account. If you do not already have an account, contact [!DNL Oracle NetSuite](https://www.NetSuite.com/portal/company/contactus.shtml).
+* Have an active subscription to any product.
```
Comment (carlojavier) [inline help/sources/connectors/marketing-automation/oracle-netsuite.md]: I think we can revise this to: "An active subscription to any [!DNL Oracle NetSuite] product." Or should it be "... any [!DNL Oracle] product."

File: help/sources/connectors/marketing-automation/oracle-netsuite.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+* [!DNL Oracle NetSuite] uses OAuth 2.0 to communicate with [!DNL Oracle NetSuite] API's. 
+    * You will need your **Account ID**. Refer to the [Finding your NetSuite Account ID](https://docs.oracle.com/en/cloud/saas/netsuite/ns-online-help/section_1498754928.html#Finding-Your-NetSuite-Account-ID)] page if you do not know your NetSuite Account ID.
+    * You will need your **[!DNL NetSuite] Account ID**. Refer to the [Finding your NetSuite Account ID](https://docs.oracle.com/en/cloud/saas/netsuite/ns-online-help/section_1498754928.html#Finding-Your-NetSuite-Account-ID)] page if you do not know your NetSuite Account ID.
+    * To access the [!DNL Oracle NetSuite] API's, You will need a **Client ID** and **Client secret**, follow the [OAuth 2.0](https://docs.oracle.com/en/cloud/saas/NetSuite/ns-online-help/chapter_157769826287.html#OAuth-2.0) tutorial within your [!DNL Oracle NetSuite] account and ensure you execute the below:
```
Comment (carlojavier) [inline help/sources/connectors/marketing-automation/oracle-netsuite.md]: replace "... the below:" with "... the following:"

File: help/sources/connectors/marketing-automation/oracle-netsuite.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+        * Ensure your [!DNL Oracle NetSuite] administrator has enabled the OAuth 2.0 feature
+        * Setup Roles
+        * Assigned Users to the Oauth 2.0 Roles, and created Integration records
```
Comment (carlojavier) [inline help/sources/connectors/marketing-automation/oracle-netsuite.md]: We can revise some of these to sentence-case:
- Ensure your [!DNL Oracle NetSuite] administrator has enabled the OAuth 2.0 feature.
- Set up roles.
- Assign users to OAuth 2.0 roles and create integration records.

File: help/sources/connectors/marketing-automation/oracle-netsuite.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+    * You will also need to generate an **Access token** and a **Refresh Token**. Refer to the [OAuth 2.0 Authorization Code Grant Flow](https://docs.oracle.com/en/cloud/saas/netsuite/ns-online-help/section_158074210415.html#OAuth-2.0-Authorization-Code-Grant-Flow) tutorial to generate them. Complete [Step One](https://docs.oracle.com/en/cloud/saas/netsuite/ns-online-help/section_158081944642.html#Step-One-GET-Request-to-the-Authorization-Endpoint) and [Step Two](https://docs.oracle.com/en/cloud/saas/netsuite/ns-online-help/section_158081952044.html#Step-Two-POST-Request-to-the-Token-Endpoint) from the tutorial.
```
Comment (carlojavier) [inline help/sources/connectors/marketing-automation/oracle-netsuite.md]: - we can refer to "access token" and "refresh token" in sentence case in this context
- sentence case for both "step one" and "step two"

File: help/sources/connectors/marketing-automation/oracle-netsuite.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+| Refresh token | The Refresh token value value is generated at the end of [Step Two](https://docs.oracle.com/en/cloud/saas/netsuite/ns-online-help/section_158081952044.html#Step-Two-POST-Request-to-the-Token-Endpoint) of the [OAuth 2.0 Authorization Code Grant Flow](https://docs.oracle.com/en/cloud/saas/netsuite/ns-online-help/section_158074210415.html#OAuth-2.0-Authorization-Code-Grant-Flow) tutorial. Refresh tokens have a longer validity and are valid for 7 days. Refresh tokens are used by the [!DNL Oracle NetSuite] source to internally generate new Access tokens. | `eyJr......dmxM`<br> the value is a 1024 characters string formatted as a JSON Web Token (JWT). |
+| accessTokenUrl | The [!DNL NetSuite] Access Token URL. This takes the form shown alongside. | <code>https://<NETSUITE_ACCOUNT_ID>.suitetalk.api.netsuite.com<br>/services/rest/auth/oauth2/v1/token</code> |
```
Comment (carlojavier) [inline help/sources/connectors/marketing-automation/oracle-netsuite.md]: sentence case for mentions of "step two"

File: help/sources/connectors/marketing-automation/oracle-netsuite.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+>[!IMPORTANT]
+>
+> After a Refresh token expires you must create a New Account in Platform with updated tokens.
```
Comment (carlojavier) [inline help/sources/connectors/marketing-automation/oracle-netsuite.md]: sentence case for "refresh token"

File: help/sources/tutorials/api/create/marketing-automation/oracle-netsuite-activities.md [hunk 8]
```diff
@@ -0,0 +1,646 @@
...
+  -H 'x-api-key: {API_KEY}' \
+  -H 'x-gw-ims-org-id: {ORG_ID}' \
+  -H 'x-sandbox-name: {SANDBOX_NAME}'
+```
+
+**Response**
+
+A successful response returns a JSON structure like the following: 
+
+```json
```
Comment (carlojavier) [inline help/sources/tutorials/api/create/marketing-automation/oracle-netsuite-activities.md:143]: Let's wrap this response in a collapsable (+++) and add "Select to view JSON payload" as a prompt

File: help/sources/tutorials/api/create/marketing-automation/oracle-netsuite-entities.md [hunk 9]
```diff
@@ -0,0 +1,995 @@
...
+
+>[!NOTE]
+>
+>Some records have been truncated to allow for a better presentation.
+
+>[!BEGINTABS]
+
+>[!TAB Customer]
+
+A successful response returns a structure as below.
```
Comment (carlojavier) [inline help/sources/tutorials/api/create/marketing-automation/oracle-netsuite-entities.md:171]: Let's wrap these responses in a collapsable as well (+++), they should work within the tabs

File: help/sources/tutorials/api/create/marketing-automation/oracle-netsuite-entities.md [hunk 10]
```diff
@@ -0,0 +1,995 @@
...
+                "unbilledorderssearch": "0",
+                "weblead": "F"
+            }
+        },
+    ]
+},
+```
+
+>[!TAB Contact]
+
```
Comment (carlojavier) [inline help/sources/tutorials/api/create/marketing-automation/oracle-netsuite-entities.md:486]: Let's add a collapsable here

File: help/sources/tutorials/ui/create/marketing-automation/oracle-netsuite-activities.md [hunk 11]
```diff
@@ -0,0 +1,98 @@
...
+| Refresh token | The Refresh token value value is generated at the end of [Step Two](https://docs.oracle.com/en/cloud/saas/netsuite/ns-online-help/section_158081952044.html#Step-Two-POST-Request-to-the-Token-Endpoint) of the [OAuth 2.0 Authorization Code Grant Flow](https://docs.oracle.com/en/cloud/saas/netsuite/ns-online-help/section_158074210415.html#OAuth-2.0-Authorization-Code-Grant-Flow) tutorial. Refresh tokens have a longer validity and are valid for 7 days. Refresh tokens are used by the [!DNL Oracle NetSuite Activities] source to internally generate new Access tokens. | `eyJr......dmxM`<br> the value is a 1024 characters string formatted as a JSON Web Token (JWT). |
+| accessTokenUrl | The [!DNL NetSuite] Access Token URL. This takes the form shown alongside. | https://<ACCOUNT_ID>.suitetalk.api.netsuite.com/services/rest/auth/oauth2/v1/token |
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/marketing-automation/oracle-netsuite-activities.md]: Let's reformat the placeholder values in the examples from "<EXAMPLE>" to "{EXAMPLE}" please apply throughout the other docs too

**Review by carlojavier — CHANGES_REQUESTED** [review]
@mohan-h thanks for the submission! I've left some comments for you

File: help/sources/connectors/marketing-automation/oracle-netsuite.md (hunk 4 above)
Comment (mohan-h) [reply help/sources/connectors/marketing-automation/oracle-netsuite.md]: Have merged 1 & 2

**Pull request merged**